project/scripts/release

```diff
@@ -168,8 +168,6 @@ fi
 declare -r version=$1
 declare -r publish_path="${release_server}:${release_path}"
 
-[[ `java -version 2>&1 | grep -E "java version|openjdk version" | cut -d ' ' -f3 | cut -d '.' -f2` -eq 8 ]] || fail "Java version is not 1.8"
-
```

Comment:
Should we fail if it is not 9 though instead?

Reply:
Yes, and we do - it's a couple of lines up IIRC